src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -0,0 +1,145 @@
+﻿<vm:BasePropertiesPage
+	x:Class="Files.App.Views.Properties.HashesPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:models="using:Files.Backend.Models"
+	xmlns:toolkitconverters="using:CommunityToolkit.WinUI.UI.Converters"
+	xmlns:vm="using:Files.App.ViewModels.Properties"
+	DataContext="{x:Bind HashesViewModel, Mode=OneWay}"
+	Loaded="BasePropertiesPage_Loaded"
+	mc:Ignorable="d">
+
+	<vm:BasePropertiesPage.Resources>
+		<ResourceDictionary>
+			<toolkitconverters:BoolNegationConverter x:Key="BoolNegationConverter" />
+			<toolkitconverters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
+		</ResourceDictionary>
+	</vm:BasePropertiesPage.Resources>
+
+	<Grid x:Name="RootGrid">
```

Comment:
I think this grid is extra, the StackPanel can be move to the HashesListGird.

Reply:
HashesListGird has border and background(like a card). Even though I make ListView have border and background, I cannot put UnavailableTextBlock inside ListView. so not possible.

Reply:
👍